Sources/StreamChat/Models/Attachments/ChatMessageImageAttachment.swift

```diff
@@ -65,7 +65,7 @@ public struct ImageAttachmentPayload: AttachmentPayload {
     /// Creates `ImageAttachmentPayload` instance.
     ///
     /// Use this initializer if the attachment is already uploaded and you have the remote URLs.
-    @available(*, deprecated, renamed: "init(title:imageRemoteURL:imagePreviewRemoteURL:originalWidth:originalHeight:extraData:)")
```

Comment:
Why did we remove this?

Reply:
There was a typo, the init in the deprecation was basically mentioning the deprecated one instead of the new one 👍